.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
It was in the PR because I did not sync changes from askql to my fork of askql repository -> it treated this file as not merged to master while it was already merged in xFAANG repo.

Reply:
Either way, I have synced my fork with xFAANG and it is now gone in the changed files list.